config/routes.rb

```diff
@@ -767,6 +767,8 @@
 
     match '/stem-resources/:type/:id(/:slug)' => 'home#stem_resources', :as => :stem_resources
 
+    match '/stem-resources/:filter_name/:filter_value' => 'home#stem_resources', :as => :stem_resources
+
```

Comment:
I think this might break lightbox rendering of the form:
/stem-resources/activity/10
The slug is optional. And I'm guessing the route system matches the most specific form which in this case is probably the new one you added. So then in this case the params passed to the stem_resources method will be:
filter_name: activity
filter_value: 10

To be more safe I'd suggest just having one route instead of two. Perhaps it would be:
`'/stem-resources/:type/:id_or_filter_value(/:slug)'`

Reply:
> I think this might break lightbox rendering of the form:
> /stem-resources/activity/10

:id is matched against a regex:

```
                                             stem_resources          /stem-resources/:type/:id(/:slug)(.:format)                                                   home#stem_resources {:id=>/\d+/}

```

But I will make the changes you suggest. 

Also I am seeing /slug in the form of .slug instead. That seems like a bug in the lightbox launching code.

Reply:
Oh I didn't see that regex on the id because it was off to the right.  I think the change is still good though. That way if the home page wants to handle urls with a integer in the second part that will still be ok.